### DISCUSSION TITLE
Copy settings.db from Windows to Linux

### DISCUSSION THREAD

**Discussion by qupear** [FreeTubeApp/FreeTube#5272] [General]
Hi. Where is stored a file settings.db on Linux machine? Or how distractions free settings could be copied other way? General settings are easy to manage but there are too much of hidden channels to do copy by hand.
Found answer in documentation.

Same question for **Android**.

**Comment by kommunarr** [comment]
For Linux, to my knowledge, it should be in `~/.config/FreeTube`. Not sure about Android.

**Comment by qupear** [comment]
For Android devs added a setting to change data directory https://github.com/MarmadileManteater/FreeTubeAndroid/issues/299
